### PR TITLE
Update gpupgrade global help output

### PR DESCRIPTION
**Pull request by bhuvnesh2703**
Update help output for 
- global help
- initialize
- execute
- finalize

### CODE REVIEW

File: cli/commands/commands.go [hunk 1]
```diff
@@ -782,64 +782,63 @@ func stopHubAndAgents(tryDefaultPort bool) error {
...
 	initializeHelp = `
-Runs through pre-upgrade checks and prepares the cluster for upgrade.
+Runs pre-upgrade checks and prepares the cluster for upgrade.
 
-Initialize will carry out the following sub-steps:
+Initialize will carry out the following steps:
 %s
 
-Usage: gpupgrade initialize <flags>
+During or after gpupgrade initialize, you may revert the cluster to its
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:790]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 2]
```diff
@@ -782,64 +782,63 @@ func stopHubAndAgents(tryDefaultPort bool) error {
...
-Required Flags:
-
-  --source-gphome         path for the source Greenplum installation
+Usage: gpupgrade initialize --file <path/to/config_file>
 
-  --target-gphome         path for the target Greenplum installation
+Required Flags:
 
-  --source-master-port    the master port for the source Greenplum installation
+  -f, --file      config file containing upgrade parameters
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:797]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 3]
```diff
@@ -858,43 +857,44 @@ Optional Flags:
   -v, --verbose   outputs detailed logs for revert
 `
 	GlobalHelp = `
-gpupgrade enables users to do an in-place cluster upgrade to the next major version.
-The default mode is copy, which creates a copy of the primary segments and performs the upgrade on the copies.
+gpupgrade performs an in-place cluster upgrade to the next major version.
+
+NOTE: Before running gpupgrade, you must prepare the cluster. Refer to
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:862]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 4]
```diff
@@ -858,43 +857,44 @@ Optional Flags:
...
+  1. initialize   runs pre-upgrade checks and prepares the cluster for upgrade
 
-                  Usage: gpupgrade initialize <flags>
+                  Usage: gpupgrade initialize --file </path/to/config_file>
 
                   Required Flags:
-                    --source-gphome        path for the source Greenplum installation
-                    --target-gphome        path for the target Greenplum installation
-                    --source-master-port   the master port for the source Greenplum installation
+                    -f, --file   config file containing upgrade parameters
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:874]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 5]
```diff
@@ -858,43 +857,44 @@ Optional Flags:
...
-                    --source-master-port   the master port for the source Greenplum installation
+                    -f, --file   config file containing upgrade parameters 
+                                 (e.g. gpupgrade_config)
 
-                  Optional Flags:
-                    --mode [copy|link]     Upgrade mode to either copy source files to target or use hard links to modify data in place. Default is copy.
-                    --temp-port-range      the set of ports to use when initializing the target cluster
-                    --hub-port             the port gpupgrade hub uses to listen for commands on
-                    --agent-port           the port gpupgrade agent uses to listen for commands on
+  2. execute      upgrades the master and primary segments to the target
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:877]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 6]
```diff
@@ -858,43 +857,44 @@ Optional Flags:
...
+                    -f, --file   config file containing upgrade parameters 
+                                 (e.g. gpupgrade_config)
 
-                  Optional Flags:
-                    --mode [copy|link]     Upgrade mode to either copy source files to target or use hard links to modify data in place. Default is copy.
-                    --temp-port-range      the set of ports to use when initializing the target cluster
-                    --hub-port             the port gpupgrade hub uses to listen for commands on
-                    --agent-port           the port gpupgrade agent uses to listen for commands on
+  2. execute      upgrades the master and primary segments to the target 
+                  Greenplum version
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:878]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 7]
```diff
@@ -858,43 +857,44 @@ Optional Flags:
...
-                  Optional Flags:
-                    --mode [copy|link]     Upgrade mode to either copy source files to target or use hard links to modify data in place. Default is copy.
-                    --temp-port-range      the set of ports to use when initializing the target cluster
-                    --hub-port             the port gpupgrade hub uses to listen for commands on
-                    --agent-port           the port gpupgrade agent uses to listen for commands on
+  2. execute      upgrades the master and primary segments to the target 
+                  Greenplum version 
 
-  2. execute      upgrades the master and primary segments to the target Greenplum version
+  3. finalize     upgrades the standby master and mirror segments to the target
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:880]: Please remove the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 8]
```diff
@@ -782,64 +782,63 @@ func stopHubAndAgents(tryDefaultPort bool) error {
...
-  -v, --verbose            outputs detailed logs for initialize
 `
 	executeHelp = `
 Upgrades the master and primary segments to the target Greenplum version.
 
-Execute will carry out the following sub-steps:
+Execute will carry out the following steps:
 %s
 
+During or after gpupgrade execute, you may revert the cluster to its
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:813]: Please delete the extra <space> at the end of this line.

File: cli/commands/commands.go [hunk 9]
```diff
@@ -782,64 +782,63 @@ func stopHubAndAgents(tryDefaultPort bool) error {
...
+
 `
 	finalizeHelp = `
 Upgrades the standby master and mirror segments to the target Greenplum version.
 
-Finalize will carry out the following sub-steps:
+Finalize will carry out the following steps:
 %s
 
+Once you run gpupgrade finalize, you may not revert the cluster to its
```
Comment (dkrieger-pvtl) [inline cli/commands/commands.go:831]: Please delete the extra <space> at the end of this file.

**Comment by bhuvnesh2703** [comment]
@dkrieger-pvtl fixed the comments, also a blank line is required at the start and end, i confirmed it.

**Pull request merged**